runners/google-cloud-dataflow-java/worker/build.gradle

```diff
@@ -71,6 +71,10 @@ def excluded_dependencies = [
         library.java.truth                           // Test only
 ]
 
+// For Java8+ and less than Java11, use versions 2.x.x.
+// For Java11+ use versions 3.x.x per https://github.com/ben-manes/caffeine.
+def caffeine_cache_version = project.javaVersion < "11" ? "2.9.3" : "3.1.8"
```

Comment:
This won't work for release. The version string is resolved during compile time, so the released dataflow-worker will always get 2.9.3 (because the artifacts were compiled using single java version - java8 then publish to maven)

Reply:
removed the branch, defaulting to the lower version